pkg/controller/migration/convert/network.go

```diff
@@ -198,6 +200,40 @@ func handleCalicoCNI(c *components, install *operatorv1.Installation) error {
 		install.Spec.CalicoNetwork.HostPorts = &hp
 	}
 
+	type TuningSpec struct {
+		Sysctl []operatorv1.Sysctl `json:"sysctl,omitempty"`
+		Type   string              `json:"type"`
+	}
+
+	// CNI tuning plugin
+	if pluginData, ok := c.cni.Plugins["tuning"]; ok {
+		// parse JSON data
+		var tuningSpecData TuningSpec
+		if err := json.Unmarshal([]byte(pluginData.Bytes), &tuningSpecData); err != nil {
+
+			return ErrIncompatibleCluster{
+				err:       "error parsing CNI config plugin type 'tuning'",
+				component: ComponentCNIConfig,
+				fix:       "fix CNI config",
+			}
+		}
+
+		sysctlTuning := []operatorv1.Sysctl{}
+		for _, setting := range tuningSpecData.Sysctl {
+			if _, allowed := utils.AllowedSysctlKeys[setting.Key]; allowed {
```

Comment:
There should be an else statement for this because having an unallowed Key should result in the migration being blocked/error.  This is because if we cannot duplicate the configuration that was present in the cluster before the migration then we want to block and not proceed with the migration.

Reply:
I'd suggest just putting all settings into `sysctlTuning` and then call VerifySysctl on the slice.

Reply:
ok, changed